README.md

```diff
@@ -1,2 +1,261 @@
 # operator-certification-operator
-A Kubernetes operator to provision the OpenShift Operator Pipeline.
+A Kubernetes operator to provision resources for the operator certification
+# Requirements
+The certification operator requires that you have the following tools installed, functional, and in your path.
+- [Install](https://docs.openshift.com/container-platform/4.8/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli) oc, the OpenShift CLI tool (tested with version 4.7.13)
+- [Install](https://tekton.dev/docs/cli/) tkn, the Tekton CLI tool (tested with version 0.19.1)
+- [Install](https://git-scm.com/downloads) git, the Git CLI tool (tested with 2.32.0)
+- The certification pipeline expects you to have the source files of your Operator bundle
+
+# Pre - Installation
+### Install Openshift pipelines operator
+* Log into your cluster's OpenShift Console with cluster admin privileges
+* Use the left hand menu to navigate to *Operators*
+* In the *Operators* submenu click on *OperatorHub*
+* Use the Filter/Search box to filter on *OpenShift Pipelines*
+* Click the *Red Hat OpenShift Pipelines* tile
+* In the flyout menu to the right click the *Install* button near the top
+* On the next screen "Install Operator" scroll to the bottom of the page and click *Install*
```

Comment:
Small nit: Should these sentences include punctuation at the end?

Reply:
This is copy/paste from the existing pipeline docs, if we change here we should raise a PR to update those IMO.

Reply:
Gotcha, I am fine with it in that case